Web/JSAssignment1/js/Registration.js

```diff
@@ -0,0 +1,56 @@
+function activateSubmit() {
+
+	if (document.getElementById("acceptCondition").checked == true) {
+		document.getElementById("submitButton").disabled = false;
+
+	} else {
+		document.getElementById("submitButton").disabled = true;
+	}
+}
+
+
+function validate() {
+	var count = 0;
+	if (document.getElementById("firstName").value.trim() == "") {
+		count = 1;
+		alert("First Name Is Empty");
+	}
+	if (document.getElementById("lastName").value.trim() == "") {
+		count = 1;
+		alert("Last Name Is Empty");
```

Comment:
alerting just too many times (here if all the entries are invalid) gives a bad user experience. Either collect all the error messages and in the last show then in just one alert or show one error at a time, e.g. if name is invalid then show alert only for that therefore skipping other validations.